docs/configure-arquillian.md

```diff
@@ -0,0 +1,95 @@
+
+1. The ability to set `arquillian.xml` container properties directly within the `pom.xml`.
+2. Automatic generation of `arquillian.xml` when using the Arquillian Liberty Managed container. 
+
+For more convenient integration with the Arquillian Liberty Managed/Remote containers and JUnit/TestNG, you can use this goal with the [Arquillian Liberty Dependency Bundles](https://github.com/wasdev/arquillian-liberty-dependencies).
+
+**Notice:** Be aware of false positive test results when using certain versions of TestNG with Arquillian. More information can be found [here](https://github.com/WASdev/arquillian-liberty-dependencies/blob/master/docs/testng-false-positives.md).
+
+#### Automatic generation of `arquillian.xml`
+
+For both Arquillian Liberty Managed and Remote containers, the `configure-arquillian` goal can automatically generate and configure the `arquillian.xml` that usually exists in the `src/test/resources` directory, eliminating the need for the user to manually create the file in projects with only a single Arquillian container. 
+
+Furthermore, when using the Arquillian Liberty Managed container, `configure-arquillian` will also automatically specify the three required configuration parameters: `wlpHome`, `serverName`, and `httpPort`. These are set to values specified in the `liberty-maven-plugin`. 
+
+The `configure-arquillian` goal will perform configuration for the Liberty Managed or Remote container based on which container exists on the classpath. Arquillian does not allow for more than one container on the classpath. In the event of neither container existing on the classpath, the `configure-arquillian` goal will default to providing configuration for the Liberty Managed container. 
+
+**Common configuration parameters for the Arquillian Liberty Managed and Remote configurations:**
+
+| Property | Type | Description | Default |
+-----------| ------------ | ------- | ------- |
+| `arquillianProperties` | Dictionary | Used to set key/value pairs of configuration parameters in `arquillian.xml`. | **Managed:** A dictionary containing values for `wlpHome`, `serverName`, and `httpPort` as specified in the `liberty-maven-plugin`.<br>**Remote:** An empty dictionary when using the Arquillian Liberty Remote container. |
+| `skipIfArquillianXmlExists` | Boolean | Skips the `configure-arquillian` goal if `arquillian.xml` already exists in the `target` directory. | False |
+
+TODO: Add link to JBoss documentation of configuration parameters. These links might be outdated due to container name change...
+
+https://docs.jboss.org/author/display/ARQ/WLP+V8.5+-+Managed
+
+https://docs.jboss.org/author/display/ARQ/WLP+V8.5+-+Remote
+
+#### Usage Example
+
+Specify the following `liberty-maven-plugin` configuration in `pom.xml`:
+
+```
+<plugin>
+	<groupId>net.wasdev.wlp.maven.plugins</groupId>
+	<artifactId>liberty-maven-plugin</artifactId>
+	<version>2.1.2</version>
```

Comment:
So this will get updated to 2.2 when we put out the new release right?

Reply:
Yes. I'll update it to 2.2-SNAPSHOT now.